code/enrichment/sldsc_enrichment.ipynb

```diff
@@ -992,8 +1012,10 @@
     "            cols <- paste0(\"ANNOT_\", annot_index) \n",
     "            num <- num + ((nrow(dat)-1) * sapply(dat[, ..cols], var))\n",
     "        } else {\n",
-    "            if((4 + annot_index) > ncol(dat)) stop(paste(\"Index out of range:\", m))\n",
-    "            num <- num + ((nrow(dat)-1) * var(dat[, 4+annot_index, with=FALSE]))\n",
+    "            # Newly changed: if((6 + annot_index) > ncol(dat)) stop(paste(\"Index out of range:\", m))\n",
```

Comment:
it is okay to remove all the `#Newly changed` hints since I can read the patch for it here to keep track. Please remove them.